x/evmutil/types/conversion_pair.go

```diff
@@ -87,3 +92,82 @@ func validateConversionPairs(i interface{}) error {
 
 	return pairs.Validate()
 }
+
+///////////////
+// SDK -> EVM
+///////////////
+
+// NewAllowedNativeCoinERC20Token returns an AllowedNativeCoinERC20Token
+func NewAllowedNativeCoinERC20Token(
+	sdkDenom, name, symbol string,
+	decimal uint32,
+) AllowedNativeCoinERC20Token {
+	return AllowedNativeCoinERC20Token{
+		SdkDenom: sdkDenom,
+		Name:     name,
+		Symbol:   symbol,
+		Decimal:  decimal,
```

Comment:
Decimal must fit into a uint8 (https://eips.ethereum.org/EIPS/eip-20) so we should validate it's `< 256`

Reply:
ah yup. i'll just make it a uint8 like in the erc20 standard.
will also pluralize 👍